pkg/boilerplate/controller/filter.go

```diff
@@ -67,3 +67,17 @@ func FilterByNames(parentFunc ParentFunc, names ...string) ParentFilter {
 		DeleteFunc: has,
 	}
 }
+
```

Comment:
We treat `pkg/boilerplate` like a library, should not add arbitrary funcs to it.  We have a JIRA to bump it:
https://jira.coreos.com/browse/CONSOLE-1497

Do we need this to live somewhere?

Reply:
The above JIRA references doing an update by extracting our `pkg/boilerplate` and pulling it in as a dependency, where it now lives:
https://github.com/enj/go